pontoon/base/migrations/0096_add_entity_value_and_properties.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 4.2.22 on 2025-07-13 08:58
+
+from moz.l10n.formats.fluent import fluent_parse_entry
+from moz.l10n.message import message_to_json
+from moz.l10n.model import CatchallKey, SelectMessage
+
+from django.db import migrations, models
+
+from pontoon.sync.utils import parse_pontoon_message
+
+
+def set_value_and_properties(apps, schema_editor):
+    Entity = apps.get_model("base", "Entity")
+    Resource = apps.get_model("base", "Resource")
+
+    fluent_entities = Entity.objects.filter(resource__format="fluent")
+    for entity in fluent_entities:
+        try:
+            fe = fluent_parse_entry(entity.string, with_linepos=False)
+            entity.value = message_to_json(fe.value)
+            entity.properties = {
+                name: message_to_json(msg) for name, msg in fe.properties.items()
+            } or None
+        except Exception:
+            if entity.obsolete and entity.resource.project.disabled:
+                entity.value = [entity.string]
```

Comment:
This is needed as a workaround for some obsolete strings in the disabled test-pilot-website project on pontoon.mozilla.org, which use an out-of-date (2017) Fluent syntax representation.